test/test_util.py

```diff
@@ -102,26 +154,15 @@ def test_writeArrayToRaster(tmp_path):
     assert np.allclose(band.ReadAsArray(), array)
 
 
-def test_makePoints1D_Python(make_points_1d_data):
-    from RAiDER.utilFcns import makePoints1D
+def test_makePoints0D_cython(make_points_0d_data):
+    from RAiDER.makePoints import makePoints0D
```

Comment:
We should put the imports at the top of the file. I guess I forgot to do this when I was refactoring before

Reply:
I would use `as` to differentiate between the cython and python versions

```python
from RAiDER.makePoints import makePoints0D as make_points_0d_cython
```

Reply:
@Askaholic I completely removed the python versions of these codes as I don't think they're needed, they were only used when the cython versions weren't working.